frontend/src/components/ui/skeletons/InputSkeleton.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Skeleton } from "../skeleton";
+
+const InputSkeleton = () => {
+  return (
+    <div>
```

Comment:
You can remove this div and just return the skeleton. Try and think about when you  need divs or not. It takes up space on the browser and could cause lag if there are too many elements. In our case its fine but in general its something to think about when coding.